plugins/scaffolder-backend-module-gitlab/package.json

```diff
@@ -36,6 +36,8 @@
     "@backstage/integration": "workspace:^",
     "@backstage/plugin-scaffolder-node": "workspace:^",
     "@gitbeaker/node": "^35.8.0",
```

Comment:
This can be removed, right? I would assume that there are `yarn.lock` changes, which would need to be commmited :)

Reply:
Yes, this could be removed and i have added the yarn lock file